src/test/java/org/fcrepo/importexport/integration/ImporterIT.java

```diff
@@ -279,6 +279,33 @@ public void testImportMappedTrailingSlash() throws Exception {
         assertTrue(resourceExists(childURI));
     }
 
+    @Test
+    public void testRepositoryRoot() throws Exception {
```

Comment:
These seem more like unit tests than integration tests. Would you mind moving them to ImporterTest.java?